build issues with loadgen.cc

```
inference/loadgen/loadgen.cc: In member function ‘void mlperf::loadgen::PerformanceSummary::ProcessLatencies()’:
inference/loadgen/loadgen.cc:617:13: error: ‘struct mlperf::loadgen::PerformanceResult’ has no member named ‘query_intervals’
  617 |   assert(pr.query_intervals.size() == query_count);
      |             ^~~~~~~~~~~~~~~
```

--- a/loadgen/loadgen.cc
+++ b/loadgen/loadgen.cc
@@ -614,7 +614,6 @@ void PerformanceSummary::ProcessLatencies() {
   // Calculate per-query stats.
   size_t query_count = pr.queries_issued;
   assert(pr.query_latencies.size() == query_count);
-  assert(pr.query_intervals.size() == query_count);
   std::sort(pr.query_latencies.begin(), pr.query_latencies.end());
   QuerySampleLatency accumulated_query_latency = 0;
   for (auto latency : pr.query_latencies) {

